components/admin_console/message_export_settings.jsx

```diff
@@ -81,15 +81,15 @@ export default class MessageExportSettings extends AdminSettings {
                     id='exportJobStartTime'
                     label={
                         <FormattedMessage
-                            id='admin.messageExport.exportJobStartTime.title'
-                            defaultMessage='Message Export Time:'
+                            id='admin.complianceExport.exportJobStartTime.title'
+                            defaultMessage='Compliance Export Time:'
                         />
                     }
-                    placeholder={Utils.localizeMessage('admin.messageExport.exportJobStartTime.example', 'E.g.: "02:00"')}
+                    placeholder={Utils.localizeMessage('admin.complianceExport.exportJobStartTime.example', 'E.g.: "02:00"')}
                     helpText={
-                        <FormattedMessage
-                            id='admin.messageExport.exportJobStartTime.description'
-                            defaultMessage='Set the start time of the daily scheduled message export job. Choose a time when fewer people are using your system. Must be a 24-hour time stamp in the form HH:MM.'
+                        <FormattedHTMLMessage
+                            id='admin.complianceExport.exportJobStartTime.description'
+                            defaultMessage='Set the start time of the daily scheduled compliance export job. Choose a time when fewer people are using your system.<br />Must be a 24-hour time stamp in the form HH:MM. For example, 1:15am would be expressed as 01:15, while 1:15pm would be expressed as 13:15.'
```

Comment:
`<br />Must be a 24-hour time stamp in the form HH:MM. For example, 1:15am would be expressed as 01:15, while 1:15pm would be expressed as 13:15.` --> `Must be a 24-hour time stamp in the form HH:MM.`

I.e. no need for an example (the placeholder has an example) nor a need for a new line